config/jsdoc/plugins/api.js

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
It looks like this file still contains some stuff that is specific to OpenLayers (`olx.js`)... It'd be nice to cleanup a bit so and fit it to what we need. Although it doesn't look something very straight forward.

Reply:
Super cleanup done!

Reply:
🔝 !